contracts/true-currencies/TokenController.sol

```diff
@@ -82,6 +82,8 @@ contract TokenController {
     bytes32 public constant IS_MINT_RATIFIER = "isTUSDMintRatifier";
     bytes32 public constant IS_REDEMPTION_ADMIN = "isTUSDRedemptionAdmin";
 
+    address public proofOfReserveEnabler;
```

Comment:
Looks like you didn't move this quite high enough. There are still constants mixed in with the storage variables here. Could you move this between `gasRefunder` and `IS_MINT_PAUSER`?

Reply:
Done.